backend/src/main/java/ch/puzzle/okr/models/KeyResult.java

```diff
@@ -0,0 +1,186 @@
+package ch.puzzle.okr.models;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+public class KeyResult {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO, generator = "sequence_keyResult")
+    @NotNull
+    private Long id;
+
+    @NotNull
+    @ManyToOne
+    private Objective objective;
+
+    @NotBlank
+    @Size(min = 3, max = 20)
```

Comment:
Als Minimallänge würde ich überall 2 nehmen.
Der Maximalwert auf 250 Zeichen. In den Mockups sind teilweise lange Title enthalten.